### DISCUSSION TITLE
The page you are looking for isn't here.

### DISCUSSION THREAD

**Discussion by RJGhugo** [mastodon/mastodon#29965] [General]
Who can I redirect the user away from the error page?

![error page](https://github.com/mastodon/mastodon/assets/41795683/4a6fe5a0-9958-4716-8d80-0fa2d667c8db)

**Comment by StrangeMinds1960** [comment]
Not sure if I've understood your question correctly, but you should be able to create a custom 404 page on your web server

**Comment by RJGhugo** [comment]
Thanks.

You mean I have to do it on the proxy server and not in the application it self?

**Reply by StrangeMinds1960** [reply]
Having looked at my own Mastodon setup, I'm not sure how / where you'd do it.  Where would you want to redirect users anyway?

**Comment by RJGhugo** [comment]
Accessing the Mastodon Instance works fine. Only delete post are landing on the error page.

I would like to redirect them to the root (example.com/xxx --> example.com).

Mastodon Log:
I, [2024-04-16T18:10:25.601575 #17]  INFO -- : [...???...] method=GET path=/xxxxx format=html controller=ApplicationController action=raise_not_found status=404 allocations=959 duration=2.14 view=0.97 db=0.00

**Comment by RJGhugo** [comment]
Thanks you shall open a official request.
